### ISSUE TITLE
no database

### ISSUE THREAD

**Issue by Heimdall-Midgard** [brian8544/LegionWoW#2]
it lacks SQL database

**Comment by brian8544** [comment]
True, no real work has ever been put into this anymore. However you can use this: https://github.com/brian8544/WoW_CMS/blob/master/_INSTALL%26Stuff/sql/cms.sql
